quickstart.rb

```diff
@@ -63,9 +63,16 @@ def init_service
 
 
 first_filter = cli.ask("What filter do you want ot run?  ") { |q| q.default = "[\+|0-9][0-9|\s|\\-|a-z|A-Z]*" }
+replacement_pattern = cli.ask("Choose replacement pattern (optional)  ") { |q| q.default = "\\0" }
```

Comment:
Could think about a good query statement for the replacement pattern 🤷‍♂️

Reply:
The default replace can be an empty one, which would do the trick, Right?

This way a no-op is guaranteed

Reply:
When adding "\\\\0" as the replacement, gsub method replaces the matching expression with itself (expression with index=0 flag). I repeated '\\' twice to include '\\' as a character in the default pattern.